### PR TITLE
Task5

### PR DESCRIPTION
**Pull request by Regis25**

### CODE REVIEW

File: src/main/java/org/jalasoft/BankClient.java [hunk 1]
```diff
@@ -25,9 +25,14 @@ public void run() {
...
         scanner = new Scanner(System.in);
         while (!done) {
-            System.out.print("Enter command (0=quit, 1=new, 2=select, 3=deposit, 4=loan, 5=show, 6=interest): ");
+            System.out.print("Enter command (0=quit, 1=new, 2=select, 3=deposit, 4=loan, 5=show, 6=interest, 7=Withdraw): ");
             int commandNumber = scanner.nextInt();
-            processCommand(commandNumber);
+            
+            try {
+                processCommand(commandNumber);
+            } catch (IllegalArgumentException exception) {
```
Comment (uabarahona) [inline src/main/java/org/jalasoft/BankClient.java:33]: We still need to add ArithmeticException  here :)

**Review by uabarahona — APPROVED** [review]

**Review by uabarahona — CHANGES_REQUESTED** [review]
There is still a ArithmeticException that is not being controlled

File: src/main/java/org/jalasoft/BankClient.java (hunk 1 above)
Comment (Regis25) [reply src/main/java/org/jalasoft/BankClient.java:33]: I put it in the Bank Service line 104. Is it necessary to put again the exception or should I move that exception to this code line?

Comment (uabarahona) [reply src/main/java/org/jalasoft/BankClient.java:33]: got it, it is ok :)

**Review by uabarahona — APPROVED** [review]

**Pull request merged**